libs/positions/src/lib/positions-table.tsx

```diff
@@ -293,40 +267,45 @@ export const PositionsTable = forwardRef<AgGridReact, Props>(
         <AgGridColumn
           headerName={t('Liquidation price (est)')}
           field="liquidationPrice"
-          flex={2}
-          headerTooltip={t(
-            'Liquidation prices are based on the amount of collateral you have available, the risk of your position and the liquidity on the order book. They can change rapidly based on the profit and loss of your positions and any changes to collateral from opening/closing other positions and making deposits/withdrawals.'
-          )}
+          type="rightAligned"
+          cellRendererSelector={(): CellRendererSelectorResult => {
+            return {
+              component: PriceFlashCell,
+            };
+          }}
           filter="agNumberColumnFilter"
           valueGetter={({
             data,
           }: VegaValueGetterParams<Position, 'liquidationPrice'>) => {
-            return !data
+            return data?.liquidationPrice === undefined || !data
               ? undefined
               : toBigNum(
-                  data?.liquidationPrice,
+                  data.liquidationPrice,
                   data.marketDecimalPlaces
                 ).toNumber();
           }}
-          cellRendererSelector={(
-            params: ICellRendererParams
-          ): CellRendererSelectorResult => {
-            return {
-              component: params.node.rowPinned ? EmptyCell : ProgressBarCell,
-            };
+          valueFormatter={({
+            data,
+          }: VegaValueFormatterParams<Position, 'liquidationPrice'>):
+            | string
+            | undefined => {
+            if (!data) {
+              return undefined;
+            }
+            return addDecimalsNormalizeNumber(
+              data.liquidationPrice,
+              data.marketDecimalPlaces
+            );
           }}
-          valueFormatter={progressBarValueFormatter}
         />
         <AgGridColumn
           headerName={t('Leverage')}
           field="currentLeverage"
           type="rightAligned"
           filter="agNumberColumnFilter"
-          cellRendererSelector={(
-            params: ICellRendererParams
-          ): CellRendererSelectorResult => {
+          cellRendererSelector={(): CellRendererSelectorResult => {
```

Comment:
suggestion (non blocking): we can just use the regula cellRenderer as we always render the PriceFlashCell. Same elsewhere here..